modules/ROOT/pages/traversal-framework/bidirectional_traversal_framework.adoc

```diff
@@ -0,0 +1,81 @@
+:description: The Neo4j Bidirectional Traversal Framework Java API.
+
+:org-neo4j-graphdb-bidirectional-traversal-description: {neo4j-javadocs-base-uri}/org/neo4j/graphdb/Transaction.html#bidirectionalTraversalDescription()
+
+[[Bidirectional-Traversal-Framework]]
+= The Bidirectional Traversal Framework
+
+A bidirectional traversal consists of two traversals starting from different nodes, returning the paths where both traversals collide.
+
+The Bidirectional Traversal Framework allows the description of such a traversal with the link:{org-neo4j-graphdb-bidirectional-traversal-description}[BidirectionalTraversalDescription^].
```

Comment:
Shouldn't these words be separated with spaces? 
```suggestion
The Bidirectional Traversal Framework allows the description of such a traversal with the link:{org-neo4j-graphdb-bidirectional-traversal-description}[Bidirectional Traversal Description^].
```

Reply:
This is the class name and it links to the class java docs